tests/util.test.ts

```diff
@@ -105,6 +105,52 @@ describe('Util', () => {
       });
     });
   });
+
+  describe('#stringToInt', () => {
```

Comment:
Is it possible to get a test that returns NaN?

Reply:
Done